metadata_updater/metadata_updater.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/python3
+################################################################################
+#
+# Copyright 2018 Crown copyright (c)
+# Land Information New Zealand and the New Zealand Government.
+# All rights reserved
+#
+# This program is released under the terms of the new BSD license. See the
+# LICENSE file for more information.
+#
+################################################################################
+
+import koordinates
+import os
+import sys
+import fileinput
+import yaml
+import re
+import requests
+import logging
+
+try:
+    import log
```

Comment:
log module doesn't appear to be used? And I don't think it was in the setup.py?

Reply:
Log module was not pushed. now tested

Reply:
Thanks